src/obsolete-reporter.js

```diff
@@ -0,0 +1,64 @@
+const fs = require("fs");
+const path = require("path");
+
+const TOUCHED_FILE_LIST_PATH = path.join(
+  process.cwd(),
+  ".jest-image-snapshot-touched-files"
+);
+
+const IS_ENABLED = !!process.env.JEST_IMAGE_SNAPSHOT_TRACK_OBSOLETE;
+
+class ObsoleteReporter {
+  static isEnabled() {
+    return IS_ENABLED;
+  }
+
+  static markTouchedFile(filePath) {
+    if (!IS_ENABLED) return;
+    const fd = fs.openSync(TOUCHED_FILE_LIST_PATH, "as");
```

Comment:
`fd`? Can we use a more friendly name?

Reply:
`fd` is fairly common for a file descriptor in my experience, but I can understand it's opaque for those that haven't seen the pattern before.

`touchedListFileDescriptor`?